### DISCUSSION TITLE
Difficulty finding how to subscribe

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#13843] [Mobile]
Using the GitHub 1.57.0 app on iOS, I subscribed to a few individual issues in a repository by clicking the ellipsis button at the top right corner and choosing "Subscribe" from the menu. I then wanted to subscribe to all issues in that repository, navigated to the repository page, and clicked the ellipsis button there, but the pop up menu had no "Subscribe" option in it. It took me a while to find that I had to click the bell button at the right side of the star button, instead.

![image](https://user-images.githubusercontent.com/46201428/160775343-0e5e1909-a4a3-4fa4-849d-18e51bf12e24.png)

If the menu had included a "Subscribe" item that does the exact same thing as the bell button, then it would have been easier for me to find. Perhaps it would have confused some other user, though.

**Comment by brianlovin** [comment]
Thanks for the feedback, I can see how it's confusing to have a similar function placed in different locations for the repo and individual issues/pulls. I'll pass this along to the team and see what makes the most sense as far as duplicating placement.
